cluster-autoscaler/core/scale_down_node_processor.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
```

Comment:
I think this should live under processors/, not core/.
We made an exception for filterOutSchedulablePodListProcessor, but IIRC it was only because it needed stuff from core/ and it was the easiest way of dealing with cyclic dependencies.

Reply:
Done